elastic/kibana
# [Discover] Switch documents correctly in `Document Explorer`

## Summary

Fixes #133830

### Test
1. Create two documents with the same `_id` in different indices:
```
POST index1/_doc/1
{
  "message": "hello"
}

POST index2/_doc/1
{
  "message": "hello"
}
```
2. Add a new "Data View" that can see both indices (`index*`)
3. Click on any of the two documents in the Document Explorer, they should be displayed correctly. 

### Before

https://user-images.githubusercontent.com/17125/172467910-5df1602c-9998-4764-a913-6f91076fcb04.mov


### Checklist
